src/plugins/home/public/application/components/explore_experience.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import {
+  EuiButton,
+  EuiCallOut,
+  EuiCodeBlock,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiModal,
+  EuiModalBody,
+  EuiModalFooter,
+  EuiModalHeader,
+  EuiModalHeaderTitle,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+
+interface ConfigNoticeModalProps {
+  onClose: () => void;
+}
+
+export const ExperienceSelectionModal = ({ onClose }: ConfigNoticeModalProps) => {
+  return (
+    <EuiModal onClose={onClose} maxWidth={800}>
+      <EuiModalHeader>
+        <EuiModalHeaderTitle>OpenSearch Dashboards Configuration Notice</EuiModalHeaderTitle>
```

Comment:
Shall we wrap these texts with i18n?

Reply:
will update UI components with i18n in the next PR after @kgcreative 's review